client/my-sites/media-library/test/data-source.jsx

```diff
@@ -36,14 +36,15 @@ describe( 'MediaLibraryDataSource', () => {
 					/>
 				</ReduxProvider>
 			);
-			expect( wrapper.find( 'button[data-source="google_photos"]' ) ).toHaveLength( 1 );
-			expect( wrapper.find( 'button[data-source="pexels"]' ) ).toHaveLength( 1 );
+
+			expect( screen.queryByText( 'Google Photos' ) ).toBeInTheDocument();
```

Comment:
It seems like with these changes, we no longer need to pass `data-source` to the `Popover` mock?